core/src/test/java/org/jboss/jbossset/bugclerk/utils/FixVersionHelperTest.java

```diff
@@ -65,7 +65,7 @@ public static JiraIssue createIssueWithType(IssueType issueType) {
     }
 
     public static JiraIssue createIssueMock() {
-        JiraIssue mock = MockUtils.mockJiraIssue(bugId, "A Summary...");
```

Comment:
I would rather keep mockJiraIssue over mockJira, as the first one is more clear (we mock an issue coming from Jira, not all JIRA server).

Reply:
This way it seem more inline with mockBug imho.

Reply:
Well, I would rather refactor mockBug into mockBzIssue in a separate commit. Would mind removing this part of your changes for now ?